utils.go

```diff
@@ -255,7 +255,11 @@ func (s *Storage) formatFileObject(v oss.ObjectProperties) (o *typ.Object, err e
 	o = s.newObject(false)
 	o.ID = v.Key
 	o.Path = s.getRelPath(v.Key)
-	o.Mode |= typ.ModeRead
+	if v.Type == "Symlink" {
```

Comment:
As `v.Type`  indicates the object type, we need remove the following assignment:

```
if v.Type != "" {
    o.SetContentType(v.Type)
}
```

Reply:
Got it!